src/client/Client.cc

```diff
@@ -8558,6 +8559,11 @@ loff_t Client::_lseek(Fh *f, loff_t offset, int whence)
     ceph_abort();
   }
 
+  if (f->pos < 0) {
+    f->pos = old_pos;
+    return -EINVAL;
+  }
```

Comment:
it's better to use a local variable to store the new position. update f->pos after validating the new position

Reply:
Thanks，I optimized it。